### ISSUE TITLE
Wrong icon in devices page

### ISSUE THREAD

**Issue by Tontze** [flatsiedatsie/domoticz-aurora-theme#31]
Under devices, milight bulp icon is bulp when its not on, when turning on, it changes its icon to switch icon.
![milight_bulp](https://user-images.githubusercontent.com/28216916/36922476-9b764bde-1e70-11e8-828c-56d6f343ca9d.PNG)

**Comment by xztraz** [comment]
Does it differ from standard skins?

**Comment by Tontze** [comment]
Yes, sry. When turned on, it should simply change its color to yellow.

**Comment by xztraz** [comment]
i think it's because the icon in main menue is switched from a bulb to a switch. you could just remove the lightbulb.png under aurora/images. but then you get a lightbulb in the main menu

**Comment by xztraz** [comment]
or add a off switch icon and name it lightbulboff.png

**Comment by xztraz** [comment]
not sure it worked just adding the light off icon. still a switch icon.
